# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/arrayvec/Cargo.toml
# Commit: ef5dc3e04e5f
# Full Hash: ef5dc3e04e5f271eea0636ab3a495e95cc912f1d
# Author: Dzmitry Malyshau <dmalyshau@mozilla.com>
# Date: 2021-09-04 09:40:24
# Regressor Bug: 1726626
# File Overlap Count: 3
# Description:
#   Bug 1726626 - Move gfx/wgpu into a 3rd party dependency r=jgilbert,bholley
#   
#   This update makes wgpu a vendored dependency instead of having it in gfx/wgpu.
#   
#   ## Notes
# ==============================================================================

diff --git a/third_party/rust/arrayvec/Cargo.toml b/third_party/rust/arrayvec/Cargo.toml
--- a/third_party/rust/arrayvec/Cargo.toml
+++ b/third_party/rust/arrayvec/Cargo.toml
@@ -8,23 +8,23 @@
 # If you believe there's an error in this file please file an
 # issue against the rust-lang/cargo repository. If you're
 # editing this file be aware that the upstream Cargo.toml
 # will likely look very different (and much more reasonable)
 
 [package]
 edition = "2018"
 name = "arrayvec"
-version = "0.5.2"
+version = "0.7.1"
 authors = ["bluss"]
 description = "A vector with fixed capacity, backed by an array (it can be stored on the stack too). Implements fixed capacity ArrayVec and ArrayString."
 documentation = "https://docs.rs/arrayvec/"
 keywords = ["stack", "vector", "array", "data-structure", "no_std"]
 categories = ["data-structures", "no-std"]
-license = "MIT/Apache-2.0"
+license = "MIT OR Apache-2.0"
 repository = "https://github.com/bluss/arrayvec"
 [package.metadata.docs.rs]
 features = ["serde"]
 
 [package.metadata.release]
 no-dev-version = true
 tag-name = "{{version}}"
 [profile.bench]
@@ -51,13 +51,10 @@ version = "0.1.4"
 version = "0.1"
 
 [dev-dependencies.serde_test]
 version = "1.0"
 
 [build-dependencies]
 
 [features]
-array-sizes-129-255 = []
-array-sizes-33-128 = []
 default = ["std"]
 std = []
-unstable-const-fn = []